third_party/cc/repos.bzl

```diff
@@ -3,20 +3,18 @@
 # SPDX-License-Identifier: Apache-2.0
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 def cc_repos():
-    git_repository(
+    http_archive(
```

Comment:
Can you make sure all of these have hashes?

Reply:
fixed.